### ISSUE TITLE
If the mesh doesn't include tangents, the water/wave effect doesn't work

### ISSUE THREAD

**Issue by kring** [CesiumGS/cesium-unreal#524]
After #499, we only generate tangents for glTFs with a normal map. But that means that the water effect now looks shiny but not wavy:
![image](https://user-images.githubusercontent.com/924374/123920945-d9c2ac80-d9c9-11eb-8a09-9238b907cdb1.png)

The wave effect is implemented as a normal manipulation via a custom material (i.e. the glTF itself doesn't have a normal map), so this isn't too surprising, but I hadn't realized it until now.

There are two possible solutions that I know of:

1. Automatically enable tangent generation when the water mask is enabled.
2. Set `M_CesiumOverlayWater` to use world-space normals instead of tangent space:

![image](https://user-images.githubusercontent.com/924374/123921220-1db5b180-d9ca-11eb-8413-0f3e4fa43cf6.png)

The trouble with option 2 is that it makes the "normal" pin on the material output node be in world space rather than tangent space, which in turn makes the "no normal map" texture that we use when we don't have a real normal map be interpreted in world space. So that breaks lighting. I think if we could just not output to the normal pin at all when there's no normal map we'd be fine, but I don't know of a way to do that (other than by adding separate materials for with/without a normal map).

Long story short, I think we just need to enable tangent generation when the water effect is enabled.
